src/lib/components/layout/Grid/helpers.js

```diff
@@ -0,0 +1,19 @@
+export const generateResponsiveCustomProperties = (prop, infix) => {
```

Comment:
I would prefer to have one named export per file where the the filename is the name of the function. So this file should be named `helpers/generateResponsiveCustomProperties.js`

It makes the implementation, but especially the tests much easier to read.